docs/manual/annotating-libraries.tex

```diff
@@ -537,8 +537,12 @@
 a warning whenever a stub file specification is redundant with
 bytecode annotations.
 
-A file being compiled takes precedence over a stub file.  If file \<A.java>
-is being compiled, then any stub for class \<A> is ignored.
+By default, a file being compiled takes precedence over a stub file:  if
```

Comment:
This partially duplicates, but also contracts, the paragraph above.  Please fix that so that one paragraph does not state a fact unequivocally that the next retracts.

Reply:
I do not think that the paragraph above states any fact that this paragraph retracts. I have clarified that "a file being compiled" must be source code for this paragraph to apply. The previous paragraph only discusses bytecode.

If you are referring to some other contradiction, please be specific about what it is so that I can clarify it.

Reply:
Could you to say from the outset that there are three possible locations for information, rather than presenting them as two pairwise comparisons?  I found that confusing.

Reply:
I've modified the writing to address your concern. Can you take another look?